libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt

```diff
@@ -28,57 +32,69 @@ internal object LoggerProvider {
 /**
  * Alias of [LoggerProvider.logger]#v
```

Comment:
there and below correspondently d-> i ->w -> e
```suggestion
 * Alias [com.mapbox.common.Logger.v]
```